Game/Game.pde

```diff
@@ -15,17 +19,23 @@ void setup() {
   //noCursor();
 }
 
+void createGameObjects() {
+
+  Thing a = thingFactory.createWall(4, 3);
+  board.put_on_board(a, a.p, a.r, 2, 3);
```

Comment:
Incorrect use of put_on_board. If the shape is 1x1, use put_on_board(a, a.p, a.r) or put_on_board(a, a.p, a.r, 1, 1)

Reply:
True. This was just for testing if it works to have objects on more than one tile. Normally the wall object should only be placed one tile at a time